cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable require-jsdoc */
+class IframePage {
+  private urlPage: string;
+  private titleSelect: string;
+  private cssPagebutton: string;
+
+  constructor() {
+    this.urlPage = "https://www.w3schools.com/html/html_iframe.asp";
+    this.titleSelect = "[title='W3Schools HTML Tutorial']";
+    this.cssPagebutton = "a[title='CSS Tutorial']";
+  }
+  public visit(): void {
+    cy.visit(this.urlPage);
+  }
+
+  public getFrameTitle(): any {
+    return cy.get(this.titleSelect).its("0.contentDocument").its("body").then(cy.wrap).find("h1");
```

Comment:
the idea is to use the power of the plugin we just installed, so, why not just use iframe command?: `cy.iframe(this.titleSelect ).find("h1").its(0);`